### PR TITLE
Created Ping Pong Score Keeper Hacktoberfest

### PR DESCRIPTION
**Pull request by tanishqsindhu**
# Zero To Mastery - Webblocks 2022 📦📦

 ## PR contains:
- Created a Ping Pong Score Keeper between Two Player 🤜
<!---
## Breaking changes
- None

## Screenshots
icon

### CODE REVIEW

File: src/blocks/tanishqsindhu/index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+</head>
+
+<body>
+    <section>
+        <div class="container">
+            <div class="columns">
+                <div class="column is-half is-offset-one-quarter">
+                    <div class="card">
+                        <div class="card-image">
+                            <!-- <figure class="image is-2by1">
```
Comment (sophiabrandt) [inline src/blocks/tanishqsindhu/index.html:19]: If the code is not needed, remove it please.

File: src/blocks/tanishqsindhu/index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (sophiabrandt) [inline src/blocks/tanishqsindhu/index.html:8]: Please add a more descriptive title.

File: src/blocks/tanishqsindhu/index.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                            <p class="card-header-title">
+                                Ping Pong Score Keeper
+                            </p>
+                        </header>
+                        <div class="card-content">
+                            <div class="content">
+                                <h1 class="title is-1"><span id="p1Display">0</span> to <span id="p2Display">0</span>
+                                </h1>
+                                <p class="subtitle">Use the Buttons below to kepp score</p>
+                                <label for="" class="label is-large is-inline">Playing To</label>
```
Comment (sophiabrandt) [inline src/blocks/tanishqsindhu/index.html:34]: Empty `label for` is an anti-pattern, see [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attr-for](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attr-for), please fix this.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

**Comment by tanishqsindhu** [comment]
Fixed the above-listed issues.

**Pull request merged**